config/application.rb

```diff
@@ -279,7 +279,7 @@ def dynflow
       dynflow.eager_load_actions!
       dynflow.config.increase_db_pool_size
 
-      unless dynflow.config.lazy_initialization
+      unless Rails.env.production?
```

Comment:
This is not the right condition: in production, the lazy initialization is actually false: the question, if the world will have an executor, depends on something else (search for `remote?` method). I think this condition should just go away completely

Reply:
@timogoebel I share your opinion - see https://github.com/theforeman/foreman/pull/4871

Reply:
We also need to check, whether we are in rake (uness in the worker rake), to prevent the issues with db drop, that jenkins is hitting: sry for not being more verbose: Writing at phone at the moment.